src/js/builtins/ProcessObjectInternals.ts

```diff
@@ -117,9 +121,7 @@ export function getStdinStream(fd) {
     }
   }
 
-  const tty = require("node:tty");
-
-  const ReadStream = tty.isatty(fd) ? tty.ReadStream : require("node:fs").ReadStream;
+  const ReadStream = tty.isatty(fd) ? tty.ReadStream : fs.ReadStream;
```

Comment:
```suggestion
  const ReadStream = require("node:tty").isatty(fd) ? require("node:tty").ReadStream : require("node:fs").ReadStream;
```

Reply:
This was not resolved, it is eagerly loading `node:fs` when it isn't always necessary.